### DISCUSSION TITLE
Expand the Data Visualization palette

### DISCUSSION THREAD

**Discussion by Phyele** [buildo/bento-design-system#291] [Ideas]
I'd like to broaden the data visualization palette by making these changes:
- Increase the number of colors, from 10 to 15. In doing so, some old colors may need to change their names.
- Add a Pastel palette (lighter than Soft) with intensity 5 (e.g., Red 5).
- Modify Soft colors intensity from 10 to 20.
- Modify Bright colors intensity from 30 to 40.
- Add a Dark palette with intensity 60.

Do you think there could be problems doing it? Would this involve breaking changes?

**Comment by gabro** [comment]
Adding tokens is a breaking change, since users will need to update the theme on their end, or they will get a build error.

It can be done, of course, but it would require a breaking release.

Changing the "intensity" of existing colors only concerns the default theme, so that can be done in a non-breaking way and it shouldn't affect clients.
